nodejs/awsx/ec2/vpc.ts

```diff
@@ -205,10 +233,122 @@ export class Vpc extends pulumi.ComponentResource {
      * This can be done by calling [subnet.createRoute] and passing in the newly created NatGateway.
      */
     public addNatGateway(name: string, args: x.ec2.NatGatewayArgs, opts: pulumi.ComponentResourceOptions = {}) {
-        const natGateway = new x.ec2.NatGateway(name, this, args, opts);
+        const natGateway = new x.ec2.NatGateway(name, this.parent, args, opts);
         this.natGateways.push(natGateway);
         return natGateway;
     }
+}
+
+(<any>VpcData).doNotCapture = true;
+utils.Capture(VpcData.prototype).addInternetGateway.doNotCapture = true;
+utils.Capture(VpcData.prototype).addNatGateway.doNotCapture = true;
+utils.Capture(VpcData.prototype).partition.doNotCapture = true;
+
+export class Vpc extends pulumi.ComponentResource {
+    public readonly id: pulumi.Output<string>;
+
+    public readonly vpc: pulumi.Output<aws.ec2.Vpc>;
+
+    private readonly _underlyingData: Promise<VpcData>;
+
+    constructor(name: string, args: VpcArgs | ExistingVpcArgs | ExistingVpcIdArgs, opts?: pulumi.ComponentResourceOptions);
```

Comment:
this is the same constructor that existed before.  so people can still call this normally.